src/aoapplication.cpp

```diff
@@ -20,6 +20,8 @@ AOApplication::AOApplication(QObject *parent)
     : QObject(parent)
 {
   net_manager = new NetworkManager(this);
+  debug_func = new debug_functions(this);
```

Comment:
`debug_func` is too vague as a variable name.
`debug_functions` is meaningless as a class name nor does it follow at all any kind of somewhat established naming convention.